<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTypeDialog</class>
 <widget class="QDialog" name="AddTypeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Seleção de tipo</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_11">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="newTypeButton">
       <property name="text">
        <string>Criar novo tipo</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="existingTypeButton">
       <property name="text">
        <string>Selecionar tipo existente</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="newTypeLabel">
         <property name="text">
          <string>Nome do tipo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="newTypeEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="prefixLabel">
         <property name="text">
          <string>Prefixo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="prefixEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Descrição:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriptionEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="existingTypeLabel">
     <property name="text">
      <string>Selecione tipo existente:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="typeComboBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="parentChildLabel">
         <property name="text">
          <string>Pai -&gt; Filho</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="minCardChildLabel">
           <property name="text">
            <string>Nº mínimo de filhos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minCardChildEdit">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="maxCardChildLabel">
           <property name="text">
            <string>Nº máximo de filhos</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxCardChildEdit">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="childDependencyCheckBox">
         <property name="text">
          <string>Filho é dependente?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QLabel" name="childParentLabel">
         <property name="text">
          <string>Filho -&gt; Pai</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QLabel" name="minCardParentLabel">
           <property name="text">
            <string>Nº mínimo de pais</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minCardParentEdit">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QLabel" name="maxCardParentLabel">
           <property name="text">
            <string>Nº máximo de pais</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxCardParentEdit">
           <property name="text">
            <string>*</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="parentDependencyCheckBox">
         <property name="text">
          <string>Pai é dependente?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>newTypeButton</tabstop>
  <tabstop>existingTypeButton</tabstop>
  <tabstop>newTypeEdit</tabstop>
  <tabstop>prefixEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
  <tabstop>typeComboBox</tabstop>
  <tabstop>minCardChildEdit</tabstop>
  <tabstop>maxCardChildEdit</tabstop>
  <tabstop>childDependencyCheckBox</tabstop>
  <tabstop>minCardParentEdit</tabstop>
  <tabstop>maxCardParentEdit</tabstop>
  <tabstop>parentDependencyCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddTypeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddTypeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
